### ISSUE TITLE
Pandas dataframe.query - Variable used in .query gets refactored due to it being unused

### ISSUE THREAD

**Issue by rishi1111** [sourcery-ai/sourcery#101]
### Sourcery Version

Often available in Sourcery settings
Plugin - 0.8.3

### Code editor or IDE name and version

Pycharm Plugin - 0.8.3

### OS name and version
Linux
![image](https://user-images.githubusercontent.com/17124299/99394928-bac54d80-2905-11eb-8d58-8a62fb82b08c.png)

**Comment by brendanator** [comment]
It looks like `df.query` is calling `eval` internally. Anything that evaluates string can potentially cause serious problems with our analysis. Right now I don't know how we'd go about addressing this.

There's 2 options in the meantime:
1. If there's only this single case or a small number then use a [skip comment](https://github.com/sourcery-ai/sourcery/wiki/Skipping-refactorings#skip-comments):
```python
for list_value in range(index_1, index_2): # sourcery skip # NOSONAR do not remove
    ...
```

2. If there's lots of queries where this is happening then disable this refactoring globally in the [Sourcery config file](https://github.com/sourcery-ai/sourcery/wiki/Sourcery-configuration#refactor-skip):
```yaml
refactor:
  skip:
    - for-index-underscore
```

**Comment by Hellebore** [comment]
Fixed with 0.9.3
